packages/babel-plugin-transform-parameters/test/fixtures/parameters/generator/output.js

```diff
@@ -0,0 +1,12 @@
+function fn(a, _ref) {
```

Comment:
Should the wrapper be a generator? So that `fn.constructor` still work after transpiled.

Reply:
No, because a generator would start paused. `yield* fn()` already works, because `fn()` returns an instance of `GeneratorFunction`. `fn.constructor` is not `GeneratorFunction` anymore, but there are already many cases where the prototype of a function is not correct and this is just another one.

Reply:
Another reason will be that yield expression is not allowed in formal parameters.